dist/README.md

```diff
@@ -1,37 +1,102 @@
-
-* Dex as an OIDC provider, by default only using Authelia as a backend but optionally can also use Google or Microsoft as a identity provider
+* Traefik as a reverse proxy and for TLS termination (handling HTTPS)
+* Authelia as an email + password manager, optionally with 2FA and
+  password resets
+* Dex as an OIDC provider, by default only using Authelia as a backend
+  but optionally can also use Google or Microsoft as a identity
+  provider
 
 # Quickstart
 
-First, make sure that [docker is installed](https://docs.docker.com/engine/install/).
+You will need a domain name for this Grist instance, as well as a
+username, password, and default email address for the initial admin
+Grist user.
 
-In order to set up authentication secrets and some basic, default configuration, first run
+In order to set up authentication secrets and some basic, default
+configuration, first run, for example,
 
 ```sh
-./bin/generate-secure-secrets
+USERNAME=grist \
+PASSWORD=grist \
+DEFAULT_EMAIL=gristadmin@example.com \
+GRIST_DOMAIN=grist.example.com \
+./bin/bootstrap-environment
 ```
 
-<!-- TODO: Logically separate out the crypto part of setup from the
-other bits required, such as APP_HOME_URL or persist location-->
-This will use Authelia to generate some cryptographic information needed in `config/secrets`, as well as an `.env` file in the current directory with some defaults for running the Compose image. You may wish to inspect the `.env` file to change some defaults, but leave the cryptographic choices alone. Then run as usual,
+This will use Authelia and OpenSSL to generate some cryptographic
+information needed in `persist/secrets`, as well as an `.env` file in
+the current directory with some defaults for running the Compose
+image.
+
+You may wish to inspect the `.env` file to change some defaults,
+
+```sh
+nano .env
+```
+
+but leave the cryptographic choices alone.
+
+Then run as usual,
 
 ```sh
 docker compose up
 ```
 
 This creates the following default setup:
 
-* Grist at http://grist.localhost
-<!--TODO: let user configure the default username/password -->
-* Default username: `test`
-* Default password: `test`
+* Grist at https://$GRIST_DOMAIN
+* Username for the admin user: $USERNAME
+* Password for the admin user: $PASSWORD
+* Email for the admin user: $DEFAULT_EMAIL
+
+If you want to add, modify or delete users, you can do this in
+`persist/users_database.yml`. All of the instructions needed
+to do that are in the file.
+
+Compose will start, in this order, Traefik, Authelia, Dex, and finally
+Grist.
+
+# Using systemd
+
+If everything looks fine, you may interrupt Compose (hit Ctrl-C), run
+`docker compose down` and use systemd to handle the Grist Docker process:
+
+```sh
+sudo systemd start grist
+```
+
+The Compose logs will then be available in systemd's journal service,
+for example,
+
+```sh
+sudo journalctl -xeu grist
+```
+
+This will ensure that Grist restarts cleanly if you need to stop or
```

Comment:
If I recall correctly, using "always restart" in docker-compose also brings up the containers on startup, when the docker daemon starts. Might have misremembered though!

Reply:
You're right, and that's not what I want.

I shouldn't enable the unit during VM build time. I should leave it disabled and instead instruct the user to `systemctl enable grist --now` if they want to let systemd manage the service.

I've added a fixup change to address this.